header-slim.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * The Header for our theme.
```

Comment:
Can the metadata here be updated so that the content reflects how this header is different? Something like "Simplified slim header with no Libraries site navigation"

Reply:
thanks for the comments, I have updated the header title and removed the duplicate function